src/engine/client/ClientApplication.cpp

```diff
@@ -42,6 +42,8 @@ extern SDL_Window *window;
 #endif
 #endif
 
+static Cvar::Cvar<bool> client_errorPopup("client.errorPopup", "Enable the error popup window", Cvar::TEMPORARY, true);
```

Comment:
Remove TEMPORARY? TEMPORARY, like ARCHIVE, has little reason to be used now that cvar commands don't automatically archive things.

Reply:
Removed.